src/freenet/pluginmanager/PluginManager.java

```diff
@@ -883,10 +883,13 @@ public String dumpPlugins() {
 	 * look for PluginInfo for a Plugin with given classname
 	 * @param plugname
 	 * @return the PluginInfo or null if not found
-	 * @deprecated As opposed to its JavaDoc, this function will also return plugins with a matching filename, no only class name.
-	 *             To fix this ambiguity, instead use either {@link #getPluginInfoByClassName(String)} or {@link #getPluginInfoByFileName(String)}.
+     * @deprecated As opposed to its JavaDoc, this function will also return plugins with a matching
```

Comment:
To be clear it is not necessary to review callers when making this change to documentation.